domains.txt

```diff
@@ -394,6 +439,8 @@ linuxrecruit.aeml3.co.uk
 linuxrecruit.co.uk
 logansinclair.com
 logicrec.com
+logikk.co.uk
```

Comment:
Logikk is a specialist recruitment business with a niche focus on Data Science, Advanced Analytics and Business Intelligence